blueberries.css

```diff
@@ -539,6 +548,13 @@
         border: 1px solid hsl(var(--hue), 16%, 45%);
     }
 
+    /*  Apparently an HTML "select" element cannot be properly styled in its expanded form.
+        So this exists to mitigate contrast issues with the (unstylable) selected item. */
+    .compose-form__poll-wrapper select option {
+        background-color: hsl(var(--hue), 16%, 19%);
+    /*     color: hsl(var(--hue), 17%, 15%); */
+    
```

Comment:
I think their is a `}` missing around here.

Be careful maybe more code can be missing too!

Reply:
Fixed it, thanks for letting me know! I have also looked more carefully into all other rules for potential similar issues and haven't found any other.